ivy_tests/test_ivy/test_frontends/test_paddle/test_stat.py

```diff
@@ -129,6 +129,49 @@ def test_paddle_numel(
     )
 
 
+# quantile
+@handle_frontend_test(
+    fn_tree="paddle.quantile",
+    dtype_x_and_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=1,
+        min_value=-1e10,
+        max_value=1e10,
+        valid_axis=True,
+        force_int_axis=True,  # Assuming this forces the axis to be an integer.
+    ),
+    q=st.floats(0.0, 1.0),
+    keepdim=st.booleans(),
+)
+def test_paddle_quantile(
+    *,
+    dtype_x_and_axis,
+    q,
+    keepdim,
+    on_device,
+    fn_tree,
+    frontend,
+    backend_fw,
+    test_flags,
+):    
+    input_dtypes, (x,), axis = dtype_x_and_axis
+    # Convert axis if it's a tuple
+    if isinstance(axis, tuple):
+        axis = axis[0] if len(axis) == 1 else list(axis)
```

Comment:
what's the purpose of this logic, I don't think this is required.

Reply:
Since axis varibale has to be int not tuples when accepting parameters